### PR TITLE
Add note about project being unmaintained

### PR DESCRIPTION
**Pull request by rahulporuri**
This PR adds a note about the project being unmaintained. This is being done before archiving this github repository.

### CODE REVIEW

**Review by jwiggins — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,9 @@
+This package is no longer maintained, and no current Enthought Tool Suite projects
+should rely on it. If you encounter projects that still depend on enthought/enaml,
```
Comment (kitchoi) [inline README.rst:2]: > no current Enthought Tool Suite projects should rely on it.

What about projects that are not in ETS, say kitchoi has a library that depends on enaml? Should those projects try to unplug themselves too?

Comment (rahulporuri) [reply README.rst:2]: > What about projects that are not in ETS, say kitchoi has a library that depends on enaml? Should those projects try to unplug themselves too?

That's handled by the second sentence, IIUC.

Comment (rahulporuri) [reply README.rst]: cleaned up the first sentence to just say "no current projects" instead of "no current ETS projects"

Comment (kitchoi) [reply README.rst]: Thank you! LGTM!

**Pull request merged**